bundles/org.openhab.binding.lgwebos/src/main/java/org/openhab/binding/lgwebos/internal/handler/LGWebOSConfiguration.java

```diff
@@ -43,9 +45,15 @@ public int getPort() {
         return port;
     }
 
+    public String getMacAddress() {
+        String m = macAddress;
+        return m == null ? "" : m;
+    }
+
     @Override
     public String toString() {
-        return "WebOSConfiguration [host=" + host + ", port=" + port + ", key.length=" + getKey().length() + "]";
+        return "WebOSConfiguration [host=" + host + ", port=" + port + ", key.length=" + getKey().length()
+                + ", macAddress=" + macAddress + "]";
```

Comment:
As host and macAdress are nullable, could you please replace host by getHost() and macAdress by getMacAdress() (which can't return null) to avoid a potential NPE ?

Reply:
this will just print “null” no NPE. but sure, can also use getters

Reply:
If no NPE, keep it unchanged.